### PR TITLE
fix: handles non-existing versions when listing

### PR DESCRIPTION
**Pull request by victormlg**

### CODE REVIEW

**Review by larsewi — CHANGES_REQUESTED** [review]
Kan du legge til

```
Ticket: <TICKET_NO>
Signed-off-by <EMAIL>
```
i commit message?

Prøv disse stegene:
```
cd ~/ntech/cf-remote # Gå inn i cf-remote mappen
git checkout list # Pass på at du er på riktig branch
git commit --amend -s # Endre commit å legg til signatur (i.e., -s)
git push -f origin # Overskriv origin remote 
```

**Review by larsewi — APPROVED** [review]
Works great :rocket: 
```
$ cf-remote --version 1.2.3 list
Available releases: master, 3.24.x, 3.24.0, 3.21.x, 3.21.5, 3.21.4, 3.21.3, 3.21.2, 3.21.1, 3.21.0
cf-remote: CFEngine version '1.2.3' doesn't exist (yet).
```

**Review by larsewi — CHANGES_REQUESTED** [review]
Hold on... F-strings are not available in Python 3.5. You will need to format the string in another way. E.g.:

```python
user_error("CFEngine version '%s' doesn't exist (yet)." % version)
```

**Review by larsewi — APPROVED** [review]
Thanks :rocket:

File: cf_remote/commands.py [hunk 1]
```diff
@@ -279,6 +280,9 @@ def _iterate_over_packages(tags=None, version=None, edition=None, download=False
     releases = Releases(edition)
     print("Available releases: {}".format(releases))
 
+    if version not in [rel.version for rel in releases.releases] :
```
Comment (olehermanse) [inline cf_remote/commands.py:283]:
```
if version and version not in [rel.version for rel in releases.releases]:
```

Comment (olehermanse) [reply cf_remote/commands.py:283]: https://github.com/cfengine/cf-remote/pull/97

**Pull request merged**